Gemfile.lock

```diff
@@ -216,6 +216,7 @@ DEPENDENCIES
   pbs (~> 2.2.1)
   rails (= 4.2.11)
   rails_12factor
+  redcarpet (~> 3.4.0)
```

Comment:
If you use `OodAppkit.markdown.render` then you don't need to explicitly include the redcarpet gem here.